recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/dm-salvator-x-h3.cfg

```diff
@@ -8,17 +8,16 @@ displays = (
         id = 1;
     }
 );
-
 layers = (
     {
         name = "agl";
         id = 1001;
         display = "agl";
-        width = 1080;
-        height = 1487;
+        width = 1024;
```

Comment:
Should the DM configuration be changed for M3 as well?

Reply:
Updated